account_invoice_overdue_reminder/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Overdue Invoice Reminder',
+    'version': '12.0.2.0.0',
```

Comment:
```suggestion
    'version': '12.0.1.0.0',
```
Module should be start at version 12.0.1.0.0

Reply:
yes

Reply:
Héhé, this module already had a "long" life, is already used in production ! And there are migration scripts associated with this version number (due to changes in the datamodel), so I won't change it!